src/DynamoCoreWpf/ViewModels/FileTrust/FileTrustWarningViewModel.cs

```diff
@@ -10,7 +10,7 @@ namespace Dynamo.Wpf.ViewModels.FileTrust
     public class FileTrustWarningViewModel : ViewModelBase
     {
         private bool showWarningPopup;
-        private double PointerHorizontalOffset = 15;
+        private double PointerHorizontalOffset = 60;
```

Comment:
could you add a comment saying, the number here will make the pointer stand in the middle?